README.md

```diff
@@ -137,6 +137,59 @@ output = Cyberwatch_Pyhelper().request(
 )
 ```
 
+## Cyberwatch-CLI
+
+`cyberwatch-cli` located in the `cli/` folder, is a tool allowing you to manage additional functionalities. You can learn more about them using the `help` subcommand.
+
+```bash
+python3 cyberwatch-cli.py help
+```
+
+**operating systems**
+
+You can list all the operating systems known by your Cyberwatch instance :
+
+```bash
+python3 cyberwatch-cli.py os list
+```
+
+**airgap assets**
+
+You can import all the airgap scripts to execute on your asset. If no directory is given, this will create a default `cyberwatch-airgap` folder.
```

Comment:
```suggestion
You can download all the airgap scripts to execute on your asset. If no directory is given, this will create a default `cyberwatch-airgap` folder.
```